.github/ISSUE_TEMPLATE/epic.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Epic (Internal Use)
+about: This issue is used to track a feature/project that may span several days
+  to weeks to implement. This is used by the core Metabase team to project plan
+  and manage work.
+title: "[Epic] Title"
+labels: ".Epic"
+
+---
+
+**Links**
+- product doc: _link to product doc_
+- eng doc: _link to technical design doc_
+- feature branch: `branch-name` _this should be the feature branch where this work will be done in. PRs will be delivered against this branch_
+- issue links: _list of issues this should close when this is delivered_
+
+**Implementation Plan**
+
+
+***Milestone 1***
+_insert tasklist here_
```

Comment:
Can we add the actual syntax?

So we don't need to write it down every time :)

```[tasklist]
### Tasks
- [ ] task
```

Reply:
yes, but it will not be rendered as the actual tasklist according to https://github.com/orgs/community/discussions/50502. still useful?

Reply:
Uh weird... well not very useful then